freqtrade/strategy/interface.py

```diff
@@ -338,6 +338,32 @@ def informative_pairs(self) -> ListPairsWithTimeframes:
 # END - Intended to be overridden by strategy
 ###
 
+    def gather_informative_pairs(self) -> ListPairsWithTimeframes:
```

Comment:
I would propose a 2-level approach for this method.
While `informative_pairs()` might return different values each time - the informative decorator won't.

As such - we can store the values for the decorator-level "informative" pairs after the first call - and avoid iterating throught he whole class dict every time.

Potentially, that can even be moved to strategy_resolver - so it's not dependent on having been called at least once (which means if something goes wrong, it'll fail at startup, not some time after).